youtube_dl/extractor/americastestkitchen.py

```diff
@@ -80,11 +83,10 @@ def _real_extract(self, url):
             'url': 'https://player.zype.com/embed/%s.js?api_key=jZ9GUhRmxcPvX7M3SlfejB6Hle9jyHTdk2jVxG7wOHPLODgncEKVdPYBhuz9iWXQ' % video['zypeId'],
             'ie_key': 'Zype',
             'description': clean_html(video.get('description')),
-            'timestamp': parse_iso8601(video.get('publishDate')),
+            'timestamp': unified_timestamp(video.get('publishDate')),
```

Comment:
The timestamp was not a valid iso8601 timestamp. This fixes that issue and the `timestamp` in the second test case proves it